Ошибка: HTTPConnectionPool(host='76', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x0000020276541360>: Failed to resolve '76' ([Errno 11001] getaddrinfo failed)"))